doc/api/api_changes/2017-12-26-AL.rst

```diff
@@ -0,0 +1,2 @@
+`~.tight_layout.auto_adjust_subplotpars` now raises `ValueError` instead of `RuntimeError` when sizes of input lists don't match
+````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````````
```

Comment:
What does this long line of ``````s do?

Reply:
It's just a very long title... (I don't think this entry deserves both a title *and* some contents in the section.)